contracts/src/v0.8/functions/tests/1_0_0/testhelpers/FunctionsClientWithEmptyCallback.sol

```diff
@@ -28,5 +25,7 @@ contract FunctionsClientWithEmptyCallback is FunctionsClient {
     emit SendRequestInvoked(requestId, sourceCode, subscriptionId);
   }
 
-  function fulfillRequest(bytes32 requestId, bytes memory response, bytes memory err) internal override {}
+  function fulfillRequest(bytes32 /*requestId*/, bytes memory /*response*/, bytes memory /*err*/) internal override {
+    // NOP
```

Comment:
No-op? Thought this meant node operator for a second

Reply:
Hah. Missed an O. I must have NOPs on the brain.